src/ovms.h

```diff
@@ -79,6 +79,14 @@ typedef enum OVMS_BufferType_enum {
     OVMS_BUFFERTYPE_HDDL,
 } OVMS_BufferType;
 
+typedef enum OVMS_ServableState_enum {
+    OVMS_BEGIN,
```

Comment:
For datatype and buffertype enums we specified what the enum means:
OVMS_**DATATYPE**_I32
OVMS_**BUFFERTYPE**_GPU

To keep the convention we should add STATE to this enum:
```
OVMS_STATE_LOADING,
OVMS_STATE_AVAILABLE,
```